### ISSUE TITLE
Flypeople stay hungry even after eating vomit

### ISSUE THREAD

**Issue by CynicalElephant** [tgstation/tgstation#70716]

**Comment by Time-Green** [comment]
What was your original species? Where did you get the vomit from? If from food, which foods?

Please add more info

**Comment by CynicalElephant** [comment]
Sorry about that. My original species was human. The vomit was from me consuming any food, pizza was one of the foods I ate for example. Also wheat, apples, pumpkins. No matter how much of the vomit I slurped up, my hunger bar continued to increase to red.

**Comment by Time-Green** [comment]
Alright just wanted to make sure, because some roundstart spawned vomit doesn't hold nutrients. 

Goddammit why do flypeople keep breaking

**Comment by VladinXXV** [comment]
I wasn't able to reproduce this. I joined as a human and went through the teleporter until I became a flyperson. Set my nutrition to 0. Ate a few margherita pizzas, puked, drank the puke. Eventually I got all the way back to satisfied.
![image](https://user-images.githubusercontent.com/44104681/197648750-2f86ed6e-8a8b-4f3a-b474-dafe7603c860.png)
![image](https://user-images.githubusercontent.com/44104681/197648778-d4e4a7a0-5a8d-4640-979e-f5f9f75280b3.png)
![image](https://user-images.githubusercontent.com/44104681/197648825-49b8f3ad-262f-4d58-a702-8d4fbdab742f.png)
